x-pack/plugins/uptime/common/translations.ts

```diff
@@ -21,11 +21,11 @@ export const VALUE_MUST_BE_AN_INTEGER = i18n.translate('xpack.uptime.settings.in
 export const MonitorStatusTranslations = {
   defaultActionMessage: i18n.translate('xpack.uptime.alerts.monitorStatus.defaultActionMessage', {
     defaultMessage:
-      'Monitor {monitorName} with url {monitorUrl} is {statusMessage} from {observerLocation}. The latest error message is {latestErrorMessage}',
+      'Monitor {monitorName} with url {monitorUrl} from {observerLocation} {statusMessage}. The latest error message is {latestErrorMessage}',
     values: {
       monitorName: '{{state.monitorName}}',
       monitorUrl: '{{{state.monitorUrl}}}',
-      statusMessage: '{{state.statusMessage}}',
+      statusMessage: '{{{state.statusMessage}}}',
```

Comment:
@dominiqueclarke Good point! Let me explain. I had to use three braces to unescape html special characters https://github.com/janl/mustache.js/issues/244. In the reason message we use this character `>` and since code was using Mustache, it was escaped and looked broken. 

After I separated alerts reason message from the alert connector message, I stopped using Mustache https://github.com/elastic/kibana/blob/main/x-pack/plugins/uptime/server/lib/alerts/common.ts#L60 to generate the alert reason anymore, so I can use the two braces.

Reply:
Three braces are also required for display URL values properly in slack etc.

Reply:
Yep I didn't remove three braces from display URL to not break anything there.

@shahzad31 So do you think removing three braces from statusMessage, will cause `>` not appear properly in slack? Since I am not using the generateAlertMessage that internally uses Mustache, I thought it's safe to remove it. At the moment my license doesn't allow me to select Slack connector so that I can do end to end manual testing.

Reply:
I think adding back the three braces is fine, it was just important to me that I understood why it was the way it was.

Reply:
@dominiqueclarke ok I'll bring the three braces back. Let me know if there's something else that need to be addressed in this PR. If you agree we can merge this one and if anything we can follow up in another PR